### PR TITLE
Spanish translation changes

### PR DESCRIPTION
**Pull request by AlanFitzi**
Here are some small changes, Although I'm having one problem:

On the 40 "Close", I haven't found it on the page but I'm pretty sure is not translated properly. For that, I need to know where that word or button is. I'm pretty sure the correct translation is "Cerrar" but I need to be sure of it

### CODE REVIEW

**Comment by o-alexandrov** [comment]
@AlanFitzi thank you for the help with corrections ❤️ 
The `Close` could be seen on the following screenshot:
Screen Shot 2020-09-12 at 12 17 56 PM

and here:
Screen Shot 2020-09-12 at 12 19 13 PM

**Comment by AlanFitzi** [comment]
Ok! then Close means "Cerrar" in this case

File: src/spanish.ts [hunk 1]
```diff
@@ -174,9 +174,9 @@ export const text: text = {
   youDidNotUpdateAnyValues: `No actualizaste ningún valor`,
   youHaveNotSavedAnyChanges: `No ha guardado ningún cambio.`,
   yourAccountIsVerifiedPleaseSignIn: `Su cuenta está verificada. Por favor inicie sesión`,
-  yourArts: `Tus artes`,
+  yourArts: `Tu arte`,
```
Comment (o-alexandrov) [inline src/spanish.ts:177]: Quoting Alan:
> We say `Artes`, if it's followed by `Las` which mean `The` but plural. We say `Tu Arte` when we mean it in both singular and plural. Saying `Tus Artes` is incorrect, maybe it's different for English but in Spanish we have words that can be used as plural only in certain cases.

**Comment by o-alexandrov** [comment]
Thank you very much for the translation!
I extremely apologize for the delay here; from now on, we will review translation PRs within a week!

Added (merged) your translation in https://github.com/anime-club/translations/pull/16 (keeping you as the author in commits)
We needed to have a separate PR due to a long delay from our side that resulted in file conflicts.

**Pull request closed without merging**